modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[APIRest]APIGenerator/GenerateOpenAPIFromAPIApplicationDefinition.testcase

```diff
@@ -0,0 +1,208 @@
+@component-name = "portal-headless"
+definition {
+
+	property custom.properties = "feature.flag.LPS-184413=true${line.separator}feature.flag.LPS-167253=true${line.separator}feature.flag.LPS-178642=true${line.separator}feature.flag.LPS-186757=true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Object";
+	property testray.main.component.name = "Headless Builder";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginUI();
+	}
+
+	tearDown {
+		ApplicationAPI.deleteAllAPIApplication();
+
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@priority = 5
+	test CanGenerateOpenApiFromApiApplicationDefinition {
+		property portal.acceptance = "true";
+		property test.liferay.virtual.instance = "false";
```

Comment:
Is there any specific reason we don't use the virtual instances for these tests? 🤔

Reply:
The reason I added `test.liferay.virtual.instance = "false"` is we haven't point out the tests are tested on a virtual instance in test plan ticket :smile: @magjed4289 Do we need to run all the tests on a virtual instance?